Spanner/src/Transaction.php

```diff
@@ -65,6 +65,7 @@
 class Transaction implements TransactionalReadInterface
 {
     use TransactionalReadTrait;
+    use RequestHeaderTrait;
```

Comment:
Don't need to add the RequestHeaderTrait since it's already included in TransactionalReadTrait;

Reply:
Nice catch, thanks!